### ISSUE TITLE
Not Found error when using workflow dispatch

### ISSUE THREAD

**Issue by wildlifehexagon** [peter-evans/slash-command-dispatch#78]
Hello again! I was experimenting with the `workflow_dispatch` type today and could not get it work within my repository. I pretty much just used your `hello-workflow` example to try it out, but every time I issue a `/hello-workflow` command I get a vague ` ##[error]Not Found` error. Here are my two workflows:

pr-chatops.yaml
```yaml
name: Slash Command Dispatch
on:
  issue_comment:
    types: [created]
jobs:
  slashCommandDispatch:
    runs-on: ubuntu-latest
    steps:
      - name: Dispatch workflow slash command
        id: hello
        uses: peter-evans/slash-command-dispatch@v2
        with:
          token: ${{ secrets.REPO_ACCESS_TOKEN }}
          reaction-token: ${{ secrets.REPO_ACCESS_TOKEN }}
          commands: hello-workflow
          dispatch-type: workflow
          static-args: |
            repository=${{ github.repository }}
            issue-number=${{ github.event.issue.number }}
            comment-id=${{ github.event.comment.id }}
            actor=${{ github.actor }}
      - name: Edit comment with error message
        if: steps.hello.outputs.error-message
        uses: peter-evans/create-or-update-comment@v1
        with:
          comment-id: ${{ github.event.comment.id }}
          body: |
            > ${{ steps.hello.outputs.error-message }}
```

hello-workflow-command.yaml
```yaml
name: hello-workflow-command
on:
  workflow_dispatch:
    inputs:
      repository:
        description: "The repository from which the slash command was dispatched"
        required: true
      comment-id:
        description: "The comment-id of the slash command"
        required: true
      issue-number:
        description: "The issue number in which the slash command was made"
        required: true
      actor:
        description: "The user who executed the slash command"
        required: true
jobs:
  helloWorld:
    runs-on: ubuntu-latest
    steps:
      - name: Add reaction
        uses: peter-evans/create-or-update-comment@v1
        with:
          token: ${{ secrets.REPO_ACCESS_TOKEN }}
          repository: ${{ github.event.inputs.repository }}
          comment-id: ${{ github.event.inputs.comment-id }}
          reaction-type: hooray

      - name: Create URL to the run output
        id: vars
        run: echo ::set-output name=run-url::https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID

      - name: Create comment
        uses: peter-evans/create-or-update-comment@v1
        with:
          token: ${{ secrets.REPO_ACCESS_TOKEN }}
          repository: ${{ github.event.inputs.repository }}
          issue-number: ${{ github.event.inputs.issue-number }}
          body: |
            Hello @${{ github.event.inputs.actor }}!
            [Click here to see the command run output][1]
            [1]: ${{ steps.vars.outputs.run-url }}
```

These workflows work perfectly in your demo example so I'm not sure why it keeps erroring out for me. Any ideas?

**Comment by peter-evans** [comment]
Hi @wildlifehexagon 

If I'm remembering correctly, when testing this feature I found that the `.yaml` extension doesn't work for the target command. Try changing `hello-workflow-command.yaml` to `hello-workflow-command.yml`.

**Comment by wildlifehexagon** [comment]
Yeah that did the trick. What a weird little bug. Any ideas why only `.yml` works?

**Comment by peter-evans** [comment]
Now I'm not sure if I remembered correctly in my earlier comment. I can't remember if it didn't work through the GitHub API, or if I decided to hard code `.yml` as the extension for the target.

Assuming it did work and I decided to hard code it, I would have needed to add another command input to specify the extension to support multiple options. I guess at the time I didn't really want to add another command parameter just for that, so since `.yml` is the standard which GitHub Actions uses in official documentation and examples I though it wouldn't be much of an issue to hard code it.

https://github.com/peter-evans/slash-command-dispatch/blob/master/src/github-helper.ts#L166
